README.md

```diff
@@ -83,14 +83,13 @@ Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md).
 
 (open a pull request if you want your project to be added here)
 
-- https://github.com/paritytech/polkadot
-- https://github.com/paritytech/substrate
-- https://github.com/sigp/lighthouse
-- https://github.com/golemfactory/golem-libp2p
-- https://github.com/comit-network
-- https://github.com/rs-ipfs/rust-ipfs
-- https://github.com/marcopoloprotocol/marcopolo
-- https://github.com/ChainSafe/forest
-- https://github.com/ipfs-rust/ipfs-embed
-- https://www.actyx.com/developers/
-- https://github.com/starcoinorg/starcoin
+- [COMIT](https://github.com/comit-network/xmr-btc-swap) - An open protocol facilitating trustless cross-blockchain applications.
```

Comment:
Disclaimer: I used to work on this.

The description is outdated in that the original project (comit-rs) is dead so it is a bit misleading. This project is also only community maintained but the description on that repo is more accurate: "Bitcoin–Monero Cross-chain Atomic Swap "